### ISSUE TITLE
About sub-extension name zp64

### ISSUE THREAD

**Issue by tclin914** [riscv/riscv-p-spec#33]
In risc-v isa document, it mentions "Standard extensions can also be named using a single “Z” followed by an alphabetical name and an optional version number" in Chapter 27.6. I am not sure that is a legal name "zp64". The extension with number may be confused with version number.

**Comment by chuanhua** [comment]
You are correct. The Zp64 extension will be renamed to Zpregpair to avoid the "version number confusion". Thanks.
